### ISSUE TITLE
Error: Issue does not exist or you do not have permission to see it.

### ISSUE THREAD

**Issue by fcalderon** [jgillick/jira-changelog#8]
Hi everyone, 

Thanks for this amazing project. It has been very helpful to me and my team. 

I'm facing an issue when attempting to generate a changelog.
 
Error: `UnhandledPromiseRejectionWarning: StatusCodeError: 404 - {"errorMessages":["Issue does not exist or you do not have permission to see it."],"errors":{}}`

I get the same for every single ticket. However, the tickets do exist and things were working fine 2 weeks ago, which was when I first upgraded to 1.5.0 with the new auth mode (token). 

I know the issue is probably on my end, but if you can, I will appreciate any help you can provide. 

This is what I've tried in an attempt to debug the issue.

* I tried fiddling with the configuration, basing my configuration on your sample [file](https://github.com/jgillick/jira-changelog/blob/master/changelog.config.js) in this repo
* I tried manually using the [jira-client](https://github.com/jira-node/node-jira-client), which is one of your dependencies, to see if I would get the same error. However, I was able to find and log the ticket just fine, with the exact same credentials
* I tried updating the ticket version manually using the jira-client and succeeded without any issues either

I will continue to look into this and will update this issue with my findings (if any).

**Comment by jgillick** [comment]
Hi. That's very odd and thank you for all the details; it looks like you've really dug into this.

I'd first double check that the version you're calling is still 1.5.0 (I'm sure you've probably already done this):
```bash
npx  jira-changelog --version
```

Then, go into the node console and use the jira-changelog modules directly:
```js
const Config = require('jira-changelog').Config;
const Jira = require('jira-changelog').Jira;

// Get config and load jira API
const config = Config.readConfigFile('<PATH TO REPO>/changelog.config.js');
const jira = new Jira(config);

// Search for a JIRA ticket
jira.getJiraIssue('MO-123').then(console.log, console.error);
```

It's possible that the git commit parsing is extracting the ticket number incorrectly, which is why it's returning 404.

**Comment by fcalderon** [comment]
Hi @jgillick, 

Thanks for the quick reply. I confirmed that I'm running `1.5.0` by running your suggested command (`npx  jira-changelog --version`).

I also used the jira-changelog module as you suggested, and things worked as expected, so you might be right on the money with the git commit not being parsed correctly.

But, to increase my confusion, I tried generating the changelog on a different computer and things worked as expected. Could it be the version of node, or the version of git? I'll continue to investigate on this, and since using the module worked, I don't think the issue is necessarily with `jira-changelog`. I'd like to keep this issue open either way and I'll update it with my findings and close it if you don't mind.

Versions: 

| Device | Node  | Git  | Npm  |
|:-:|:-:|:-:|:-:|
| Working computer  | 10.15.1  | 2.20.1  | 6.4.1  |
| Computer with issue  | 10.13.0  | 2.19.1 |  6.4.1 |

**Comment by fcalderon** [comment]
Update: I ended up downgrading the `node` version to `10.13.0` in the computer with issues and things worked out. I did what @jgillick suggested and things worked as expected, which makes me believe that the problem is on my end. 

That said, I'll close this issue and will update it once I figure out what's going on. If it turns out that the issue is with `jira-changelog`, I'll open another issue with steps to reproduce. 

Thanks again for your help.
